src/NodaTime.Demo/XmlSchemaDemo.NetCore.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
+using System.Xml.Schema;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using NodaTime.Xml;
+using XmlSchemaClassGenerator;
+
+namespace NodaTime.Demo
```

Comment:
If all of this is really for XmlSchemaDemo, could we make all the supporting types nested types within that? You could potentially use partial classes and keep two files (but have one "nested" in the other effectively in Visual Studio). I just want to avoid anyone getting confused at all if they're not looking for this feature.

Reply:
I have rewritten this with a partial XmlSchemaDemo class + private nested support classes to make it clear that those classes are only for the XML Schema demo.